docs/modules/plugins/pages/plugin-applitools.adoc

```diff
@@ -200,7 +200,7 @@ It's possible to perform several visual testing checks at once by passing severa
 
 [source,gherkin]
 ----
-When I run visual test with Applitools using:$applitoolsConfigurations and screenshot config:$screenshotConfiguration
+When I run visual test with Applitools using:$applitoolsConfigurations and screenshot config:$screenshotParameters
```

Comment:
Shouldn't you rename the step also? screenshot config -> screenshot parameters

Reply:
need to discuss

Reply:
done reverse between screenshotParameters and screenshotConfiguration to not touch naming